packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -750,6 +756,7 @@ class PopupMenuButton<T> extends StatefulWidget {
     @required this.itemBuilder,
     this.initialValue,
     this.onSelected,
+    this.onCancelled,
```

Comment:
we should spell this `onCanceled` (one "L")

Reply:
huh...I don't think I have ever actually spelled it correctly...  Spelling updated.

Reply:
Both are correct spelling, we've just standardised on `canceled` for consistency.